### ISSUE TITLE
Please add some binary sensors

### ISSUE THREAD

**Issue by ReliableBob** [amandel/esphome-ld2410#5]
Hello!
Thank you very much for the great work you have done!!!
Please add the following binary sensors to the code: **Presence Target**, **Moving Target** and **Still Target**...
I understand that the binary Presence sensor is tied to the GPIO and I am ready to agree, but the other two that the sensor gives via UART are also very useful. Having received their state, we can build our own additional logic...

**Comment by amandel** [comment]
Good point, thanks. Do you think we should add one with the similar behavior of the gpio? Does it add value to have 3?

**Comment by ReliableBob** [comment]
Thanks for the answer!
I think that having the state of binary sensors **Presence Target, Moving Target and Still Target,** there is no need to solder a wire to **GPIO14**. After all, you can use any **GPIO** and enable them programmatically using on_press and on_releases.
For example, when the value "False" appears on the **Presence Target** sensor, you can programmatically turn off **GPIO14**, and vice versa...
And also, I ask you to add a button that will restart the LD2410 module without resetting the current settings, if possible. Indeed, in the version with bluetooth, there is such a function in the application, but I don’t know if there is such a function in the module without bluetooth, which I use))
